packages/content-type-reaction/README.md

```diff
@@ -53,8 +55,8 @@ await conversation.messages.send(reaction, {
 });
 ```
 
-> **Note**
-> Use `contentFallback` to enable clients that don't support these content types to still display some useful context. For cases where clients *do* support these types, they can use the content fallback as alt text for accessibility purposes.
+> **Note**  
+> `contentFallback` text is provided by the codec and gives clients that _don't_ support a content type the option to display some useful context. For cases where clients *do* support the content type, they can use the content fallback as alt text for accessibility purposes.
```

Comment:
Accurate positioning of `contentFallback`?

Does line 54 reflect the content fallback text from the codec?

Reply:
yes, LGTM. we can remove L#54 as that is now handled by the codec.